src/content/docs/en/pages/secure-journey/edit-edge-firewall/protect-your-domain.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Protect your domain
+description: >-
+  Learn how to configure an Edge Firewall to protect your domain.
+meta_tags: 'secure, edge firewall, configuration, settings, domains'
+namespace: docs_guides_secure_protect_your_domain
+permalink: /documentation/products/guides/secure/edge-firewall-protect-your-domain/
+menu_namespace: secureMenu
+---
+
+
+import Tabs from '~/components/tabs/Tabs'
+import InterfaceNote from '~/includes/snippets/InterfaceNote/en/snippet.mdx'
+
+<InterfaceNote />
+
+Once you create an edge firewall with Azion, you can modify a few configurations. 
+In this guide, you'll learn how to select an edge firewall to protect a particular domain.
+
+<Tabs client:visible>
+    <Fragment slot="tab.console">Console</Fragment>
+    <Fragment slot="tab.rtm">Real-Time Manager</Fragment>
+
+<Fragment slot="panel.console">
+1. Select the **Rules Engine** tab.
+2. Click the **+ Rule Engine** button.
+3. Give a name to the rule.
+4. Select a criteria to run and catch the domain you want to run the integration on. Example: `if Hostname is equal xxxxxxxxxxxx.map.azionedge.net`.
+5. Below, select a *behavior* to the *criteria*. In this case, it'll be **Run Function**. 
+    - Select the adequate function according to the name you gave it in the instantiate step.
+6. Click the **Save** button.
+
+On the Console, you must now configure your domain so it is protected by your Edge Firewall.
+
+7. On the **Products menu**, select **Domains**.
+8. Click on the domain you want to protect with your hCaptcha function.
+9. In the **Settings** section, click on the `Edge Firewall` selector and choose the Edge Firewall you've just created.
+10. Click the **Save** button.
+
+Done. Now the integration is running for every request made to the domain you indicated.
+</Fragment>
+
+<Fragment slot="panel.rtm">
```

Comment:
This content is related to Rules Engine, covered in this guide: https://www.azion.com/en/documentation/products/guides/secure/work-with-rules-engine/

Using RTM, this process is completed in the Main Settings configurations: https://www.azion.com/en/documentation/products/guides/secure/edge-firewall-configure-main-settings/#domains-1
Here we can link to the guide and/or explain how to adjust that part.

Reply:
Fixed as per suggestions.